integration-tests/src/main/java/org/apache/polaris/service/it/env/PolarisApiEndpoints.java

```diff
@@ -39,11 +40,11 @@ public PolarisApiEndpoints(URI baseUri, String realm) {
   }
 
   public URI catalogApiEndpoint() {
-    return baseUri.resolve("api/catalog");
+    return baseUri.resolve(Path.of(baseUri.getRawPath(), "api/catalog").toString());
```

Comment:
Why bother with `Path` and `.resolve()` in this case? Why not simply concatenate the URI with `/api/catalog` and then `.normalize()` instead?

Reply:
Too many ways to do the same thing :P

But I agree that looks cleaner, updated!